CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [1.42.86](https://github.com/brave/brave-browser/releases/tag/v1.42.86)
+
+ - Added Aurora EVM chain to preloaded chains for Brave Wallet. ([#23522](https://github.com/brave/brave-browser/issues/23522))
+ - Added the ability to debounce URLs based on a regex applied to the path. ([#23121](https://github.com/brave/brave-browser/issues/23121))
+ - Added pending request notification to Wallet's icon badge for Decrypt screen. ([#23613](https://github.com/brave/brave-browser/issues/23613))
```

Comment:
Suggest updating this item to clarify it is for Brave Wallet: "Added pending request notification to Brave Wallet icon badge for decrypt screen"